### ISSUE TITLE
Plugin Verifier -  Configuration issue in LTI

### ISSUE THREAD

**Issue by aparnamichael** [OpenLiberty/liberty-tools-intellij#857]
In Plugin verifier getting a warning for LTI configuration.
Warning message - "Invalid plugin descriptor 'plugin.xml'. The plugin ID 'open-liberty.intellij' should not include the word 'intellij'."

![image](https://github.com/OpenLiberty/liberty-tools-intellij/assets/114654953/7e9a4c4d-b370-4cc2-9ba9-733664aebd87)

**Comment by aparnamichael** [comment]
When I executed the plugin verifier on my local machine, I got the following warning message:

```
The following files specified for the verification are not valid plugins:
    /Users/aparnacmichael/Documents/Devex/Codebase/liberty-tools-intellij/build/distributions/liberty-tools-intellij-24.0.6.zip
        Plugin problems:
            Invalid plugin descriptor 'plugin.xml'. The plugin ID 'open-liberty.intellij' should not include the word 'intellij'.
                This plugin problem has been reported since 2024-03-26. If the plugin was previously uploaded to the JetBrains Marketplace, it can be suppressed using the `-mute TemplateWordInPluginId` command-line switch.
```

**Comment by vaisakhkannan** [comment]
I have checked with @aparnamichael , and she confirmed that this warning is not appearing in IntelliJ versions 2024.1.5 and 2024.1.6 in the Verification Results section of the marketplace page.

I noticed something on one of the JetBrains IntelliJ Support [Forums](https://intellij-support.jetbrains.com/hc/en-us/community/posts/13362061756050-Plugin-ID-specified-in-plugin-xml-should-not-contain-intellij), they were planning to suppress this warning coming from the IntelliJ Plugin Verifier.
In one of their Plugin Verifier releases, Version 1.367, they mentioned that this warning was fixed by PR https://github.com/JetBrains/intellij-plugin-verifier/pull/1078.
